spec/src/main/resources/META-INF/smithy/bsp/extensions/scala.smithy

```diff
@@ -258,6 +258,11 @@ list ScalaTestSuiteClasses {
     member: String
 }
 
+/// TODO
```

Comment:
```suggestion
/// An empty object. The `"scala-test-suites"` `dataKind` does not need any information.
```

Reply:
What I meant is to clarify what this data kind is used for. The specification says nothing about what should happen when a server receives a request with this data kind

Reply:
edited